src/subapps/admin/components/Settings/ViewsSubView.spec.tsx

```diff
@@ -75,26 +76,19 @@ describe('ViewsSubView', () => {
     await expectRowCountToBe(3);
   });
 
-  it('shows a badge for views that have errors', async () => {
-    expect(getErrorBadgeContent(viewWithIndexingErrors)).toEqual('2');
-  });
-
-  it('does not show error badge for views that dont have errors', async () => {
-    expect(getErrorBadgeContent(viewWithNoIndexingErrors)).toBeUndefined();
-  });
-
   it('shows indexing errors when view row is expanded', async () => {
     await expandRow(viewWithIndexingErrors);
 
-    await screen.getByText(/2 Total errors/i, { selector: 'h3' });
+    await waitFor(() => {
+      const errorRows = getErrorRows();
+      expect(errorRows.length).toEqual(2);
+      screen.getByText(/2 Total errors/i, { selector: 'h3' });
 
-    const indexingErrorRows = await getErrorRows();
-    expect(indexingErrorRows.length).toEqual(2);
-
-    const errorRow1 = await getErrorRow('Mock Error 1');
-    expect(errorRow1).toBeTruthy();
-    const errorRow2 = await getErrorRow('Mock Error 2');
-    expect(errorRow2).toBeTruthy();
+      const errorRow1 = getErrorRow('Mock Error 1');
+      expect(errorRow1).toBeTruthy();
```

Comment:
Nice to have: A very trivial change that is technically not a part of this diff (so you have every right to ignore). Looking at this test, I think it can be improved. 

The `getErrorRow` function finds elements by text. Since it uses the `findBy*` query the existence of the element is [implicitly already asserted](https://testing-library.com/docs/queries/about/#types-of-queries) so, in theory, the `expect(row).toBeTruthy` assertion is not really needed.

If you prefer for the tests to be explicit and still check for the existence of the row, I think. `expect(row).toBeVisible()` is a better assertion than `toBeTruthy` because it more closely matches the real scenario (i.e. we don't really want this row to be just any JS truthy value, we want it to be a visible DOM element).